.github/workflows/android.yml

```diff
@@ -41,5 +41,9 @@ jobs:
       - name: Clean gradle cache
         run: ./gradlew clean
 
+      # Making sure the app compiles
+      - name: Build app
+        run: ./gradlew assembleProdDebug
+
       - name: Run unit tests
         run: ./gradlew testDebugUnitTest --stacktrace
```

Comment:
Didn't we need to change this command so the tests would run? Otherwise they weren't?